### PR TITLE
Add PSD Bank Berlin-Brandenburg (Bank)

### PR DESCRIPTION
**Pull request by moriarty-jim**

### CODE REVIEW

**Comment by moriarty-jim** [comment]
minimal set of required-elements not known

File: companies/psd-bb.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "psd-bb",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "PSD Bank Berlin-Brandenburg eG",
+    "address": "Datenschutz\nHandjerystr. 34–36\n12159 Berlin\nDeutschland",
```
Comment (baltpeter) [inline companies/psd-bb.json:10]: Please don't add the ATTN line to the address, as these are added automatically by our website ([see](https://github.com/datenanfragen/data#addresses)).

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record! I have requested some changes below.

**Comment by baltpeter** [comment]
> minimal set of required-elements not known

It's also ok if you submit a reasonable set that you have confirmed is enough. We usually don't know whether the set we have is actually minimal.

**Comment by moriarty-jim** [comment]
I sent a GDPR request approx. 2 years ago. It was successful with name, address and DOB. 
Should I add these as required?
I could also try a new request without DOB to possibly reduce the set of required fields. "Name" and "Address" is most likely a must-have for a bank.

**Comment by baltpeter** [comment]
> I could also try a new request without DOB to possibly reduce the set of required fields.

If you want to do that, that would be great. Otherwise, I think two years is a little too long to set `quality: tested`.

**Comment by moriarty-jim** [comment]
Then I would suggest we leave it at "verified" and I'll update it when my new request has been processed.

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**